app/src/main/java/src/Pieces/King.java

```diff
@@ -1,9 +1,15 @@
-package src;
+package src.Pieces;
 
 public class King extends Piece {
     private char figure = 'K';
     public King(boolean white, int initPosX, int initPosY) {
         super(white, initPosX, initPosY);
+        if (white == true) {
```

Comment:
it's redundant, it would be better only white because it is boolean. This happens in every chess piece.